testing/python/language/test_tilelang_language_vectorized_cast.py

```
@@ -17,15 +17,36 @@ def vectorized_cast_kernel(M: int, dtype_A: str, dtype_B: str):
 
     @T.prim_func
     def main(
-            A: T.Tensor[(M), dtype_A],  # noqa: F821
-            B: T.Tensor[(M), dtype_B],  # noqa: F821
+            A: T.Tensor[(M,), dtype_A],  # noqa: F821
```

Ruff (0.14.2)

Unused `noqa` directive (unused: `F821`)

Remove unused `noqa` directive

(RUF100)